example_scripts/download_vhi_parallel.py

```diff
@@ -0,0 +1,162 @@
+"""
+# NOTE: this file has to be run with ipython
+
+ipython example_scripts/download_vhi_parallel.py
+
+TODO:
+- add the output_dir as an argument
```

Comment:
@gabrieltseng can we use `fire` for this?

Reply:
Yes. We would probably need to accept the argument as a string, and then transform it to a pathlib.Path